measuring-client/src/main/res/layout/data_row.xml

```diff
@@ -6,5 +6,6 @@
     android:background="@drawable/list_item_background"
     android:minHeight="?android:attr/listPreferredItemHeightSmall"
     android:padding="4dp"
+    android:lineSpacingExtra="4sp"
```

Comment:
In case a line break is necessary, this makes sure the space between lines look reasonable.